tests/unit/spec/AdalSpec.js

```diff
@@ -586,6 +586,26 @@ describe('Adal', function () {
         expect(storageFake.getItem(adal.CONSTANTS.STORAGE.USERNAME)).toBeUndefined();
     });
 
+    it ('test decode with no padding', function () {
+        expect(adal._decode('ZGVjb2RlIHRlc3Rz')).toBe('decode tests');
```

Comment:
@weijjia The old test strings were all different lengths, but these are all the same length.  Is that by design?